src/main/java/gov/cms/mat/cql_elm_translation/utils/HumanReadableUtil.java

```diff
@@ -58,14 +58,23 @@ public static List<String> getMeasureTypes(Measure measure) {
   }
 
   public static String getStratification(Measure measure) {
+    // Collects and returns all stratification descriptions for display
     if (CollectionUtils.isNotEmpty(measure.getGroups())) {
+
+      StringBuilder allDescriptions = new StringBuilder();
       for (Group group : measure.getGroups()) {
         if (CollectionUtils.isNotEmpty(group.getStratifications())) {
-          return group.getStratifications().stream()
-              .map(strat -> strat.getCqlDefinition())
-              .collect(Collectors.joining("\n"));
+          allDescriptions
+              .append(
+                  group.getStratifications().stream()
+                      .map(strat -> strat.getDescription())
+                      .collect(Collectors.joining("\n")))
+              .append("\n");
         }
       }
+      if (!allDescriptions.isEmpty()) {
+        return allDescriptions.toString().trim();
+      }
```

Comment:
nice to escape for html entities for the safe side. you can use  HumanReadableUtil.escapeHtmlString
```suggestion
     return HumanReadableUtil.escapeHtmlString(allDescriptions.toString());
```